easy_biologic/base_programs.py

```diff
@@ -205,6 +205,7 @@
 from datetime import datetime as dt
 import asyncio
 from collections import namedtuple
+import logging

```

Comment:
It seems like this import isn't used.

Reply:
I believe it's used at lines 1705 and 1923.